src/frontend/package.json

```diff
@@ -23,6 +23,7 @@
     "react-dom": "^18",
     "react-redux": "^9.1.2",
     "react-textarea-autosize": "^8.5.3",
+    "sharp": "^0.33.4",
```

Comment:
Why was this dependency needed? If I recall correctly [Next](https://nextjs.org/docs/pages/building-your-application/optimizing/images) uses sharp by default.

Reply:
Yes, you are entirely correct, but for `production build`  it is necessary to have the optional Sharp package. [Link](https://www.reddit.com/r/nextjs/comments/16qx2tg/guys_can_any_one_explain_what_is_sharp_and_why/)